src/Components/Postulant/Home/postulantHome/index.js

```diff
@@ -133,69 +133,72 @@ function PostulantHome() {
           </div>
           <div className={styles.btnContainer}>
             <button onClick={EditBtn} className={styles.btn}>
-              EDIT
+              edit
             </button>
           </div>
         </div>
-        <div className={styles.divContainer}>
-          <div className={styles.cardsInfoContainer}>
-            <h2 className={styles.cardsInfoTitle}>AVAILABLE SESSIONS</h2>
-            <div>
-              {loading.sessionsLoading && (
-                <div className={styles.spinnerContainer}>
-                  <LoadingSpinner />
-                </div>
-              )}
-              {!loading && !interviews.length && (
-                <h3 className={styles.nothingHere}>No sessions available!</h3>
-              )}
-              {sessions.map((session, i) => {
-                const sessionDate = new Date(session.date);
-                const formatedSessionsDate = sessionDate.toLocaleDateString();
-                return (
-                  <div key={i} className={styles.cardsInfo}>
-                    <h3>{`${formatedSessionsDate} at ${session.time}`}</h3>
-                    <button className={styles.sessionsBtn}>TAKE</button>
+        {postulant.profiles?.length <= 0 ? (
+          <div className={styles.divContainer}>
+            <div className={styles.cardsInfoContainer}>
+              <h2 className={styles.cardsInfoTitle}>available sessions</h2>
+              <div className={styles.sessionsDiv}>
+                {loading.sessionsLoading && (
+                  <div className={styles.spinnerContainer}>
+                    <LoadingSpinner />
                   </div>
-                );
-              })}
+                )}
+                {!loading && !interviews.length && (
+                  <h3 className={styles.nothingHere}>No sessions available!</h3>
+                )}
+                {sessions.map((session, i) => {
+                  const sessionDate = new Date(session.date);
+                  const formatedSessionsDate = sessionDate.toLocaleDateString();
```

Comment:
It would be better if we use a consistent format on dates, being as DD/MM/YYYY (If 1st of February it should be displayed as 01/02/2022).

This can be achieved by adding options to `.toLocaleDateString()` as follows,

```suggestion
                  const formatedSessionsDate = sessionDate.toLocaleDateString(undefined, {
                    year: 'numeric',
                    month: '2-digit',
                    day: '2-digit'
                  });
```